index.html

```diff
@@ -49,25 +46,34 @@
         window.onNavScanCode = () => {
             console.log("nav scan code")
         };
-        */
+
         window.addEventListener('load', (e) => {
             let tryOnImage = document.getElementById("tfr-tryon-image")
+            let signout = document.getElementById("signout")
+
             let onChange = (slider, imageUrl) => {
                 console.log("slider change", slider, imageUrl)
                 tryOnImage.src = imageUrl
             }
             let slider = comps.InitImageSlider("tfr-slider", onChange)
-            let tfr = InitFittingRoom(1, "thefittingroom-modal")
+            let tfr = InitFittingRoom(1, "tfr-tryon-modal")
+            signout.addEventListener("click", (e) => {
+                tfr.user.SignOut()
+            })
             let callback = (frames) => {
-                let close = slider.Load(frames)
+                if (frames instanceof Array && frames.length > 0){
+                    let close = slider.Load(frames)
+                }
             }
-            tfr.TryOn("5003-008-14", callback)
+            document.getElementById("tryon").addEventListener("click", (e) => {
+                tfr.TryOn("5003-007-08", callback)
```

Comment:
Is this supposed to be here?

Reply:
everything in index.html is just for local testing so its fine there